sklearn/decomposition/sparse_pca.py

```diff
@@ -81,6 +81,9 @@ class SparsePCA(BaseEstimator, TransformerMixin):
            backward compatibility. It would be set to ``True`` from 0.22
            onwards.
 
+    variance : bool, optional
+        If true, this will compute the explained variance versus component.
+
```

Comment:
Please state the default value
`variance : boolean, optional (default=False)`

Reply:
done thanks!